src/persist/zcl_abapgit_persist_migrate.clas.abap

```diff
@@ -3,6 +3,7 @@ CLASS zcl_abapgit_persist_migrate DEFINITION PUBLIC CREATE PUBLIC.
   PUBLIC SECTION.
     CLASS-METHODS: run RAISING zcx_abapgit_exception.
 
+  PROTECTED SECTION.
```

Comment:
do you really need to add "PROTECTED SECTION"?

Reply:
no, SE80 added it, and next time someone edits the class via SE80 it will add it again

Reply:
Eeh, ABAP is too smart, isn't it?